src/logic/prepare/provider/parameters.ts

```diff
@@ -0,0 +1,66 @@
+import type { IntegrationParameter } from '@superfaceai/ast';
+import { prepareProviderParameters } from '@superfaceai/ast';
+import inquirer from 'inquirer';
+
+export async function selectIntegrationParameters(provider: string): Promise<{
```

Comment:
I don't like these prompts TBH.

Would this work by simply asking for optional parameter name; then exiting the loop when empty name is submitted?